packages/campaignclassic/android/build.gradle

```diff
@@ -0,0 +1,37 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.1'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 31
+//    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 31
```

Comment:
`minSdkVersion`, `compileSdkVersion` and `targetSdkVersion` for 1.x is 14, 26 and 26. Can you verify what we need to use here?